bolt-sidecar/src/state/execution.rs

```diff
@@ -229,8 +225,8 @@ impl<C: StateFetcher> ExecutionState<C> {
             // Load the default KZG settings
             kzg_settings: EnvKzgSettings::default(),
             // TODO: add a way to configure these values from CLI
```

Comment:
Is this comment still relevant?

Reply:
yes, currently only `preconf_gas_limit` is configurable

```rust
/// Other values used for validation.
#[derive(Debug)]
pub struct ValidationParams {
    pub preconf_gas_limit: u64,
    pub max_tx_input_bytes: usize,
    pub max_init_code_byte_size: usize,
}

impl ValidationParams {
    pub fn new(gas_limit: u64) -> Self {
        Self {
            preconf_gas_limit: gas_limit,
            max_tx_input_bytes: 4 * 32 * 1024,
            max_init_code_byte_size: 2 * 24576,
        }
    }
}
```

Haven't been able to review the usefulnes of the others yet, so will leave the TODO for now